apps/easypid/src/features/share/FunkeOpenIdPresentationNotificationScreen.tsx

```diff
@@ -218,7 +181,6 @@ export function FunkeOpenIdPresentationNotificationScreen() {
       trustedEntities={credentialsForRequest?.verifier.trustedEntities}
       lastInteractionDate={lastInteractionDate}
       onComplete={() => pushToWallet('replace')}
-      overAskingResponse={overAskingResponse}
```

Comment:
Why remove this?

Reply:
Left over from when ai model did not work together with proximity library